### ISSUE TITLE
Implement map in Search Results

### ISSUE THREAD

**Issue by adrien-bekker** [ibhattacharj/LocalVibes#34]
Find a map api, have it display a map with the locations of the events when searched

**Comment by Asherk2250** [comment]
this seems to be working well right now!

**Comment by ibhattacharj** [comment]
Agreed! I tried implementing it for the individual events when they are clicked, but they didn't work :( I had the key, the markers, etc etc but still nothing worked.
